docs/permission/plugin-authors/authorize-the-update-endpoint.md

```diff
@@ -0,0 +1,146 @@
+---
+id: authorize-the-update-endpoint
+title: Authorize the update endpoint
+description: Authorize the update endpoint
+---
+
+When performing updates (or other operations) on specific resources, the permissions framework allows for the decision to be based on characteristics of the resource itself. This means that it's possible to write policies that (for example) allow the operation for users that own a resource, and deny the operation otherwise.
+
+To start with, let's edit `plugins/todo-list-backend/src/service/router.ts` in a similar way as in the previous step.
```

Comment:
I wonder if we should mention this at the very beginning of the tutorial, since this might affect how the new permissions are created. Maybe we should split the tutorial in two parts: one for plugins who don't own any resource type and plugins owning their resource types?

Reply:
> I wonder if we should mention this at the very beginning of the tutorial, since this might affect how the new permissions are created.

Yep sounds good. Maybe worth a note in both places? I'm sure people will skip around, so making it more discoverable seems worthwhile.

> Maybe we should split the tutorial in two parts: one for plugins who don't own any resource type and plugins owning their resource types?

That sounds great to me. The first tutorial can be much shorter and will help people avoid unnecessarily spending time grokking the resource-related concepts.